packages/react-components/source/scss/library/components/forms/_forms.scss

```diff
@@ -89,6 +89,13 @@ $puppet-form-inline-label-gutter-width: $puppet-common-spacing-base * 12 !defaul
     @include puppet-type-h6(medium);
     font-weight: $puppet-type-weight-heavy;
   }
+
+  &.rc-form-field-label-not-visible {
+    height: 0;
+    margin: 0;
+    visibility: hidden;
```

Comment:
Is the intention to keep the label accessible but hide it visually? Looks like `visibility: hidden` will hide it from both sighted users and screen readers. https://www.w3.org/WAI/tutorials/forms/labels/#note-on-hiding-elements

Reply:
yea that was the idea--great catch